src/stacked_borrows/stack.rs

```diff
@@ -42,6 +41,11 @@ pub struct Stack {
 /// probably-cold random access into the borrow stack to figure out what `Permission` an
 /// `SbTag` grants. We could avoid this by also storing the `Permission` in the cache, but
 /// most lookups into the cache are immediately followed by access of the full borrow stack anyway.
+///
+/// It may seem like maintaining this cache is a waste for small stacks, but
+/// (a) iterating over small fixed-size arrays is super fast, and (b) empirically this helps *a lot*,
+/// probably because runtime is dominated by large stacks.
+/// arrays is super fast
```

Comment:
```suggestion
/// probably because runtime is dominated by large stacks.
```
Did GH add that back? It appeared in my suggestion somehow but I edited it out... very strange.

Reply:
Yeah, it was part of the suggestion :thinking: